diff -Naur Equalizer-1.0.2/CMake/Equalizer.spec equalizer/CMake/Equalizer.spec
--- Equalizer-1.0.2/CMake/Equalizer.spec	2011-11-09 14:07:20.000000000 -0600
+++ equalizer/CMake/Equalizer.spec	2014-03-11 17:28:48.000000000 -0500
@@ -1,5 +1,5 @@
 Name:		Equalizer
-Version:	1.0.1
+Version:	1.0.2
 Release:	1%{?dist}
 Summary:	Middleware to create and deploy parallel OpenGL-based applications
 
@@ -7,7 +7,7 @@
 License:	LGPLv2, examples are BSD licensed
 URL:		http://www.equalizergraphics.com/
 Source0:	http://www.equalizergraphics.com/downloads/%{name}-%{version}.tar.gz
-Patch0:		Equalizer-1.0.1-build-fix.patch
+Patch0:		Equalizer-1.0.2-build-fix.patch
 BuildRequires:	cmake bison flex
 BuildRequires:	boost-devel glew-devel
 BuildRequires:	libX11-devel mesa-libGL-devel
diff -Naur Equalizer-1.0.2/CMakeLists.txt equalizer/CMakeLists.txt
--- Equalizer-1.0.2/CMakeLists.txt	2013-04-01 15:38:52.000000000 -0500
+++ equalizer/CMakeLists.txt	2014-03-13 17:29:21.000000000 -0500
@@ -137,17 +137,17 @@
 if(X11_FOUND)
   if(APPLE)
     set(CMAKE_INCLUDE_SYSTEM_FLAG_CXX "-isystem ") # Disabled by GNU.cmake!?
-    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mmacosx-version-min=10.6" )
-    if(EXISTS /Developer/SDKs/MacOSX10.6.sdk)
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mmacosx-version-min=10.6 -stdlib=libstdc++" )
+    if(EXISTS /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk)
       if(EQUALIZER_PREFER_AGL)
         set(EQ_AGL_USED 1)
       else(EQUALIZER_PREFER_AGL)
         set(EQ_GLX_USED 1)
       endif(EQUALIZER_PREFER_AGL)
-    else(EXISTS /Developer/SDKs/MacOSX10.6.sdk)
+    else(EXISTS /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk)
       set(EQ_AGL_USED 1)
       set(EQ_GLX_USED 1)
-    endif(EXISTS /Developer/SDKs/MacOSX10.6.sdk)
+    endif(EXISTS /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk)
   else(APPLE)
     set(EQ_GLX_USED 1)
   endif(APPLE)
@@ -191,7 +191,7 @@
       if(_CMAKE_OSX_MACHINE MATCHES "ppc")
         set(CMAKE_OSX_ARCHITECTURES "ppc")
       else()
-        set(CMAKE_OSX_ARCHITECTURES "i386")
+#        set(CMAKE_OSX_ARCHITECTURES "i386")
       endif()
     endif()
     set(CUDA_64_BIT_DEVICE_CODE OFF) 
@@ -219,7 +219,7 @@
 set(EQ_DEFINITIONS -DEQ_BUILD_DIR="${CMAKE_BINARY_DIR}/" -DEQ_SOURCE_DIR="${CMAKE_SOURCE_DIR}/")
 if(EQUALIZER_RELEASE)
   set(EQ_DEFINITIONS ${EQ_DEFINITIONS} -DEQ_RELEASE)
-  set(CMAKE_OSX_SYSROOT /Developer/SDKs/MacOSX10.6.sdk CACHE PATH "" FORCE)
+  set(CMAKE_OSX_SYSROOT /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk CACHE PATH "" FORCE)
   if(IS_ABSOLUTE ${CMAKE_INSTALL_PREFIX})
     set(EQ_DEFINITIONS ${EQ_DEFINITIONS} -DEQ_INSTALL_DIR="${CMAKE_INSTALL_PREFIX}/")
   else()
diff -Naur Equalizer-1.0.2/libs/admin/config.cpp equalizer/libs/admin/config.cpp
--- Equalizer-1.0.2/libs/admin/config.cpp	2011-11-09 14:07:20.000000000 -0600
+++ equalizer/libs/admin/config.cpp	2014-03-11 17:28:47.000000000 -0500
@@ -15,6 +15,7 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+#include "../fabric/config.ipp"
 #include "config.h"
 
 #include "canvas.h"
@@ -61,7 +62,6 @@
 }
 }
 
-#include "../fabric/config.ipp"
 #include "../fabric/view.ipp"
 #include "../fabric/observer.ipp"
 template class eq::fabric::Config< eq::admin::Server, eq::admin::Config,
diff -Naur Equalizer-1.0.2/libs/client/CMakeLists.txt equalizer/libs/client/CMakeLists.txt
--- Equalizer-1.0.2/libs/client/CMakeLists.txt	2011-11-09 14:07:20.000000000 -0600
+++ equalizer/libs/client/CMakeLists.txt	2014-03-13 17:31:33.000000000 -0500
@@ -30,7 +30,7 @@
     set(GL_LIBRARIES "${OPENGL_LIBRARIES}")
   endif(EQ_AGL_USED)
   if(EQ_GLX_USED)
-    set(GL_LIBRARIES ${GL_LIBRARIES} GL)
+    set(GL_LIBRARIES /usr/X11R6/lib/libGL.1.dylib )
   endif(EQ_GLX_USED)
 else(APPLE)
   set(GL_LIBRARIES "${OPENGL_LIBRARIES}")
diff -Naur Equalizer-1.0.2/libs/client/config.cpp equalizer/libs/client/config.cpp
--- Equalizer-1.0.2/libs/client/config.cpp	2011-11-09 14:07:20.000000000 -0600
+++ equalizer/libs/client/config.cpp	2014-03-11 17:28:47.000000000 -0500
@@ -16,6 +16,7 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+#include "../fabric/config.ipp"
 #include "config.h"
 
 #include "canvas.h"
@@ -929,7 +930,7 @@
 }
 }
 
-#include "../fabric/config.ipp"
+//#include "../fabric/config.ipp"
 #include "../fabric/view.ipp"
 #include "../fabric/observer.ipp"
 template class eq::fabric::Config< eq::Server, eq::Config, eq::Observer,
diff -Naur Equalizer-1.0.2/libs/server/config.cpp equalizer/libs/server/config.cpp
--- Equalizer-1.0.2/libs/server/config.cpp	2011-11-09 14:07:20.000000000 -0600
+++ equalizer/libs/server/config.cpp	2014-03-11 17:28:47.000000000 -0500
@@ -16,6 +16,7 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+#include "../fabric/config.ipp"
 #include "config.h"
 
 #include "canvas.h"
@@ -1124,7 +1125,6 @@
 }
 
 #include "nodeFactory.h"
-#include "../fabric/config.ipp"
 template class eq::fabric::Config< eq::server::Server, eq::server::Config,
                                    eq::server::Observer, eq::server::Layout,
                                    eq::server::Canvas, eq::server::Node,
